sandbox/modules/identity/src/main/java/org/opensearch/identity/extensions/ExtensionSecurityConfigModel.java

```diff
@@ -0,0 +1,17 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.identity.extensions;
+
+public abstract class ExtensionSecurityConfigModel {
```

Comment:
Are you planning on implementing multiple formats for extension models or is this just in case?